status-bar/ios/Sources/StatusBarPlugin/StatusBar.swift

```diff
@@ -0,0 +1,167 @@
+import Foundation
+import Capacitor
+
+@objc public class StatusBar: NSObject {
```

Comment:
I don't think we need this `@objc`

Reply:
I was thinking this could maybe be a UIView itself - but maybe that doesn't work tying to add it on top of the WebView.